benchmarks/benchmark.py

```diff
@@ -0,0 +1,92 @@
+"""Quantifies the performance of different centroid generation techniques
+
+To measure how well a generation technique, ie. random centroids, CVT, etc,
+performs, we measure the probability of generating a random point within a
+certain region defined by the centroid of that region.
+
+The mathematical equations and foundations of this benchmarking technique is
```

Comment:
A bit grandiose; just say "The equations for this benchmark can be found in Mouret 2023: [LINK]"